### DISCUSSION TITLE
Refresh data provider AND update filter

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#6502] [Help]
In my button actions list, I'd like to refresh the data provider AND also modify the filters : not only change the value of a parameter put this parameter out of the filter.
Is there a way to do it ?
![image](https://user-images.githubusercontent.com/28592069/176479854-cb3e5e86-28de-4b00-9256-d6952d808031.png)

(my data provider is currently filtered on a specific row id  (so it contains only 1 row), and after the SAVE I want to jump to the next row ... so I want to delete the filter on ID = URL.id

**Comment by melohagan** [comment]
Hi @NicolasFabre81 

If I understand you correctly, you may be able to achieve what you want with Conditional UI. 

1. Select your Data Provider, then click *Configure conditions*.
2. Add *Update setting* condition
3. *Filtering* and define filters
4. You can then enter a condition on the right hand side.

![Screenshot 2022-07-01 at 10 41 20](https://user-images.githubusercontent.com/101575380/176869835-baed45e6-21c3-434b-a265-b428b97beb8e.png)
